Animated marker added to Symbology

selmaVH1 opened the pull request:
fixes #7468 
Animated marker added to the Style library.

DelazJ commented on docs/user_manual/style_library/symbol_selector.rst:
```diff
@@ -820,6 +821,38 @@ The magnitude of field can be scaled up or down to an appropriate size for
 viewing the field.
 
 
+.. _animated_marker:
```
```suggestion
.. index:: Animation
.. _animated_marker:
```
Let's connect sections in the index page also

selmaVH1 commented:
> @selmaVH1 I think we need to showcase this type of symbol and an animation would probably be appropriate... maybe the first one in the issue report?

@DelazJ make sense, I will add animation and go through other suggestions.

selmaVH1 commented:
@DelazJ can you, please review this PR again? I added both animations, they can be helpful. I have also removed paragraph titles, so now everything is in one paragraph, but I'm still not sure is it clear enough for users. 
Another thing, I added screenshots for PDF, but I can't create preview for PDF so I'm not sure is it ok...

DelazJ commented on docs/user_manual/style_library/symbol_selector.rst:
```diff
@@ -820,6 +821,51 @@ The magnitude of field can be scaled up or down to an appropriate size for
+* :guilabel:`File` path,
+* :guilabel:`Frame rate`: number of steps that are shown per second,
+  indicating how fast the animation is played,
+* :guilabel:`Size` in any :ref:`supported unit <supported_units>`,
```
```suggestion
* :guilabel:`Size` in any :ref:`supported unit <unit_selector>`,
```
oups, sorry!

DelazJ commented on docs/user_manual/style_library/symbol_selector.rst:
```diff
@@ -820,6 +821,38 @@ The magnitude of field can be scaled up or down to an appropriate size for
+GIFs, WebP, or MNG animations, to represent points on your map. You 
+can specify marker ``file``, ``size``, ``angle``, and ``frame rate``.
+
+There are two ways in which animated symbols are handled:
```
> I have also removed paragraph titles, so now everything is in one paragraph, but I'm still not sure is it clear enough for users.

Indeed. Maybe do we need to keep this sentence?

DelazJ commented on docs/user_manual/style_library/symbol_selector.rst:
```diff
@@ -820,6 +821,51 @@ The magnitude of field can be scaled up or down to an appropriate size for
+   :align: center
+   :width: 100%
+
+   Animated marker when map is not configured as animation
```
😮 svg? I should stop review when too late..

DelazJ commented on docs/user_manual/style_library/symbol_selector.rst:
```diff
@@ -820,6 +821,57 @@ The magnitude of field can be scaled up or down to an appropriate size for
+
+.. only:: html
+
+.. figure:: /docs/user_manual/style_library/img/animated_marker_map.gif
```
The whole block with caption should be indented, under the `only...` block. And use relative path to file. See my previous suggestion
